File: CompSci 1/Unit 6, Scanner Class/Powers/Powers.java
import java.util.Scanner;

public class Powers {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Please enter an integer --> ");
        int n = sc.nextInt();

        System.out.println("The Square = " + (n * n));
        System.out.println("The Cube = " + (n * n * n));
        System.out.println("The Fourth Power = " + (n * n * n * n));
    }
}

File: CompSci 1/Unit 6, Scanner Class/Powers/Powers.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Please enter an integer --> ")
    val n = sc.nextInt()

    println("The Square = " + (n * n))
    println("The Cube = " + (n * n * n))
    println("The Fourth Power = " + (n * n * n * n))
}